plugins/Live/templates/getLastVisitsStart.twig

```diff
@@ -18,27 +28,22 @@
                     <span class='hash'>#</span>
                     {{ visitor.goalConversions }}
                     {% if visitor.visitEcommerceStatusIcon %}
-                        &nbsp;-
+
                         <img src="{{ visitor.visitEcommerceStatusIcon }}" title="{{ visitor.visitEcommerceStatus }}"/>
                     {% endif %}
                 </span>
                 {% endif %}
                 {% if visitor.visitorTypeIcon %}
-                    &nbsp;- <img src="{{ visitor.visitorTypeIcon }}" title="{{ 'General_ReturningVisitor'|translate }}"/>
+                    <img src="{{ visitor.visitorTypeIcon }}" title="{{ 'General_ReturningVisitor'|translate }}"/>
                 {% endif %}
-                {% if visitor.visitorId|default(false) is not empty %}
-                <a class="visits-live-launch-visitor-profile rightLink" title="{{ 'Live_ViewVisitorProfile'|translate }} {% if visitor.userId is not empty %}{{ visitor.userId|raw }}{% endif %}" data-visitor-id="{{ visitor.visitorId }}">
-                    {% if visitor.userId is not empty %}<br/>{% endif %}
-                    <img src="plugins/Live/images/visitorProfileLaunch.png"/>
-                    {{ visitor.userId|default('')|raw }}
-                </a>
-                {% endif %}
-                {% if visitor.visitIp %}- <span title="{% if visitor.visitorId is not empty %}{{ 'General_VisitorID'|translate }}: {{ visitor.visitorId }}{% endif %}">
+
+                {% if visitor.visitIp %} <span title="{% if visitor.visitorId is not empty %}{{ 'General_VisitorID'|translate }}: {{ visitor.visitorId }}{% endif %}">
                     IP: {{ visitor.visitIp }}</span>
                 {% endif %}
-            </div>
             <!--<div class="settings"></div>-->
-            <div class="referrer">
+            <span style="  display: block;
+  padding-top: 4px;
+  padding-bottom: 1px;">
```

Comment:
A CSS class maybe? And probably not use a span with display block but rather a div?